loki/bulk/configure.py

```diff
@@ -179,6 +185,23 @@ def create_item_config(self, name):
             item_conf.update(self.routines[key])
         return item_conf
 
+    def create_frontend_args(self, path, default_args):
+        """
+        Create bespoke ``frontend_args`` to pass to the constructor
+        or ``make_complete`` method for a file
+
+        The resulting `dict` contains overwrites that have been provided
+        in the :attr:`frontend_args` of the config.
+        """
```

Comment:
Might be useful to list arguments here. Was briefly wondering here what `path` pertained to.